packages/headless/src/api/analytics/search-analytics.ts

```diff
@@ -153,6 +153,12 @@ export const configureAnalytics = ({
 export const getVisitorID = () =>
   new CoveoAnalyticsClient({}).getCurrentVisitorId();
 
+export const clearCoveoAnalyticsClient = () => {
```

Comment:
```suggestion
export const clearAnalyticsClient = () => {
```

Reply:
Just a preference, I'd wouldn't declare "Coveo" variables in Headless, even is coveo.analytics does